.github/workflows/build-commit.yml

```diff
@@ -0,0 +1,48 @@
+name: Build
+on: [push, pull_request]
+jobs:
+  build_amd64:
+    name: Build (AMD64 - CentOS 7 Target)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Build code-server Binary
+        run: yarn && bash ./scripts/ci.bash  
+        env:
+           VERSION: master
+           TARGET: 'linux'
+           VSCODE_VERSION: '1.38.1'
+           MINIFY: 'true' 
```

Comment:
Could we skip minify/package for non-releases? Minifying can take quite some time and I'm not sure it's useful for a non-release.